### DISCUSSION TITLE
Hottop connection with Windows Surface Laptop

### DISCUSSION THREAD

**Discussion by jlholt889** [artisan-roaster-scope/artisan#1734] [General]
Does anyone know of any issues with the connectivity between Hottop 2k+ roaster and Artisan with the new Arms processor? Im using a new Microsoft Surface Laptop, and I can't seem to connect.

**Comment by roasterdave** [comment]
Sorry, ARM64 based Windows is not currently supported by Artisan.
[Supported Platforms](https://artisan-scope.org/about/#platforms)

**Comment by GuusIJ** [comment]
I realized the connection on a Xiaomi W11 tablet bij following the instructions as described here:
https://artisan-roasterscope.blogspot.com/2015/05/hottop-kn-8828b-2k.html

You need to install the driver for serial communication over usb manually
